lasagne/layers/merge.py

```diff
@@ -91,7 +128,7 @@ def get_output_for(self, inputs, **kwargs):
             if coeff != 1:
                 input *= coeff
             if output is not None:
-                output += input
+                output = self.merge_function(output, input)
             else:
                 output = input
         return output
```

Comment:
I think this version reads more easily. We could write out the list comprehension as a for-loop but then it would be nearly the same length.